scripts/deploy.js

```diff
@@ -63,7 +63,7 @@ let pushFilesToS3 = function (config, s3) {
           ContentType: `${mimeType}`,
           ACL: 'public-read'
         }, (res) => {
-          console.log(res);
+          console.log(`successfully put ${fileName} in S3`);
```

Comment:
if we hit this block, is it always an indication of success?

Reply:
[looks like it doesn't](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property). follow up pr on they way